entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+fail () {
+    echo "$@" >&2
+    exit 1
+}
+
+shopt -s nullglob
+cvmfs_mounts=(/cvmfs/*)
+if [[ -z $cvmfs_mounts && -n $CVMFSEXEC_REPOS && -n $CVMFSEXEC_DIST ]]; then
+    echo "No cvmfs mounts; using cvmfsexec to mount $CVMFSEXEC_REPOS"
+    # We need our own copy of cvmfsexec for permissions reasons.
+    cvmfsexec_root=/tmp/cvmfsexec-$(id -u)
+    if [[ ! -d $cvmfsexec_root ]]; then
+        echo "No cvmfsexec dir found for this user at $cvmfsexec_root; creating one"
+        cp -rp /cvmfsexec $cvmfsexec_root || fail "Couldn't create $cvmfsexec_root"
+        echo "Fetching $CVMFSEXEC_DIST CVMFS config"
+        $cvmfsexec_root/makedist $CVMFSEXEC_DIST || fail "Couldn't fetch CVMFS config"
+    fi
+    $cvmfsexec_root/cvmfsexec -- /bin/true || \
+        fail "cvmfsexec smoke test failed.  You may not have the permissions to run cvmfsexec; see https://github.com/cvmfs/cvmfsexec#README for details"
+    exec $cvmfsexec_root/cvmfsexec $CVMFSEXEC_REPOS -- "$@"
```

Comment:
There should also separately be some logic in the pilot, such that if it knows that a repository needs to be mounted, it isn't mounted, and $CVMFSMOUNT is not empty, then it should invoke $CVMFSMOUNT to mount it.  Also the pilot needs to invoke `exec {CVMFSEXEC_CMDFD}>&-` if $CVMFSEXEC_CMDFD is set, to prevent giving the job access to the cvmfsexec command.

Reply:
@mambelli mmascher The GlideinWMS cvmfsexec support should check for cvmfsexec to already be running.   If it is, the pilot should skip invoking it, but still do the other things it needs to do to play well with cvmfsexec.

Reply:
(@mmascher I accidentally left out the '@' for you on the previous comment.  I edited it in but I'm not sure if github would send a notice in response to an edit, so I add this extra comment.)